wazimap/data/tables.py

```diff
@@ -603,9 +605,9 @@ def _build_model_from_fields(self, fields, table_name, geo_level=None, value_typ
 
         # create the table model
         class Model(Base):
-            table = existing_table or table_name
+            table = table_name
             __table__ = Table(table, Base.metadata, *table_args, extend_existing=True)
-        _census_table_models[table_name] = Model
+        _field_table_models[table_name] = Model
```

Comment:
I don't think this is going to work.
e.g. if we have `pop_group_by_gender` and `gender_by_pop_group` using the same table, `gender_by_pop_group` , we need both keys in `_field_table_models`. In the above only `gender_by_pop_group` will be in there.